### ISSUE TITLE
Vscode intellisense/typescript support crashes shortly after startup in the extension codebase

### ISSUE THREAD

**Issue by brad-decker** [MetaMask/metamask-extension#11137]
**Describe the bug**
Prior to #11011 we had an issue where IntelliSense would be slow, and `node:console`, `globalThis`, etc would be auto imported while typing `console.` or `this.` after #11011 those issues are no longer, but now a new one has surfaced. The TypeScript support and IntelliSense tooling completely crash shortly after startup, and you must force restart the typescript server to get it back. Once you do that the clock is ticking on how long you can benefit from VScode features before it inevitably shuts off again. 

**Steps to reproduce (REQUIRED)**
1. Be on develop
2. Start the latest version of vscode
3. Attempt to rely on autocomplete behavior to auto-import functions from others files or even node_modules
4. If it works, try again a few minutes later and see that it no longer offers any auto-complete behavior, even for variables that are in the scope of the file you're editing. 

**Expected behavior**
IntelliSense never breaks, and restarting the typescript server is only necessary when adding new methods or files that weren't previously indexed.

**Browser details (please complete the following information):**
 - OS:  Mac OS 11.3
 - VSCode Version: 1.56.2

**Additional context (Error Messages, etc.)**
Removing node_modules from the exclude array seems to have a positive impact on the rate at which IntelliSense crash, but it doesn't fully resolve the issue.

**Comment by brad-decker** [comment]
This is likely no longer relevant but I am adding my team's label for now to check on it
